doc/modules/preprocessing_targets.rst

```diff
@@ -12,7 +12,10 @@ you want to transform the prediction target for learning, but evaluate the
 model in the original (untransformed) space.
 
 Label binarization
-------------------
+==================
+
+Multiclass classification
+-------------------------
```

Comment:
sklearn considers this `'multilabel-indicator'`, i wonder if calling it "multiclass" classification would be confusing.

```py
from sklearn.utils.multiclass import type_of_target
import numpy as np
X = np.array([[1, 0, 0, 0],
              [0, 0, 0, 1]])
type_of_target(X)
# 'multilabel-indicator'
```

Reply:
You're right, that is confusing. 

I wonder if that's more a limitation of `type_of_target`, though, as `LabelBinarizer` does seem to be [meant for multiclass classification](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.LabelBinarizer.html).

Reply:
It can be used for multiclass if its One-vs-Rest. One can consider this `"mutlilabel-indicator"` where there is only one label per sample.

I think saying that this is used for one-vs-rest multiclasss classification would make it a little more clear.

Reply:
I've created #18145 to discuss this further. It might be overkill, but it could be useful for `type_of_target` to return something more specific in this case. :)